samples/react-native/package.json

```diff
@@ -28,24 +28,25 @@
     "@sentry/react-native": "6.1.0",
     "delay": "^6.0.0",
     "react": "18.3.1",
-    "react-native": "0.75.4",
-    "react-native-gesture-handler": "^2.18.1",
-    "react-native-reanimated": "3.15.0",
-    "react-native-safe-area-context": "4.10.5",
-    "react-native-screens": "3.34.0",
-    "react-native-svg": "^15.6.0",
+    "react-native": "0.76.0",
+    "react-native-gesture-handler": "^2.20.2",
+    "react-native-reanimated": "3.16.1",
+    "react-native-safe-area-context": "github:th3rdwave/react-native-safe-area-context#main",
```

Comment:
The following PR which has not been released yet is needed https://github.com/th3rdwave/react-native-safe-area-context/pull/537